README.md

```diff
@@ -11,12 +11,11 @@ This project contains the following top level components:
 
 * [api](api/): The OpenTelemetry API.
   * [trace](api/src/main/java/io/opentelemetry/trace/): The tracing api. Includes `Tracer`, `Span` and `SpanContext`.
-  * [tags](api/src/main/java/io/opentelemetry/tags/): Key-value pairs of data that can be propagated to provide contextual information.
+  * [distributedcontext](api/src/main/java/io/opentelemetry/distributedcontext/): Collection of tags in the form of key-value pairs of data that can be propagated to provide contextual information.
```

Comment:
Collection of entries? (to avoid confusion with `tags`, which was previously used).

Reply:
All right, I made the change.